collectd/collectd_test.go

```diff
@@ -14,22 +15,18 @@ import (
 type testServer string
 type serverResponses []serverResponse
 type serverResponse struct {
-	database, retentionPolicy, name string
-	tags                            map[string]string
-	timestamp                       time.Time
-	values                          map[string]interface{}
+	database        string
+	retentionPolicy string
+	points          []influxdb.Point
 }
 
 var responses = make(chan *serverResponse, 1024)
 
-func (testServer) WriteSeries(database, retentionPolicy, name string, tags map[string]string, timestamp time.Time, values map[string]interface{}) (uint64, error) {
+func (testServer) WriteSeries(database, retentionPolicy string, points ...influxdb.Point) (uint64, error) {
```

Comment:
If the majority of the time we'll always have many, then we can make it a slice.  It felt like we would be doing single point writes a lot as well from other endpoints (graphite for example).  Using vargs allows you to send in single or multiple.

Not married to it, if you have a preference, I can go either way.